packages/core/src/ui/components/Transactions/Transaction.tsx

```diff
@@ -269,6 +274,29 @@ export const Transaction = ({
             </div>
           </div>
         </div>
+
+        {deposit && (
+          <div className={styles.details}>
+            <div className={styles.title}>{t('package.core.transactionDetailBrowser.deposit')}</div>
+            <div className={styles.detail}>
+              <div className={styles.amount}>
+                <span className={styles.ada}>{`${deposit} ${coinSymbol}`}</span>
+                <span className={styles.fiat}>{amountTransformer(deposit)}</span>
+              </div>
+            </div>
+          </div>
+        )}
+        {depositReclaim && (
+          <div className={styles.details}>
+            <div className={styles.title}>{t('package.core.transactionDetailBrowser.depositReclaim')}</div>
+            <div className={styles.detail}>
+              <div className={styles.amount}>
+                <span className={styles.ada}>{`${depositReclaim} ${coinSymbol}`}</span>
+                <span className={styles.fiat}>{amountTransformer(depositReclaim)}</span>
+              </div>
+            </div>
+          </div>
+        )}
```

Comment:
I would reduce code duplication here
```suggestion
        const renderDepositValueSection = ({ value, label }) => (
          <div className={styles.details}>
            <div className={styles.title}>{label}</div>
            <div className={styles.detail}>
              <div className={styles.amount}>
                <span className={styles.ada}>{`${value} ${coinSymbol}`}</span>
                <span className={styles.fiat}>{amountTransformer(value)}</span>
              </div>
            </div>
          </div>
        )
        ...
        {deposit && renderDepositValueSection({ value: deposit, label: t('package.core.transactionDetailBrowser.deposit') })}
        {depositReclaim && renderDepositValueSection({ value: depositReclaim, label: t('package.core.transactionDetailBrowser.depositReclaim') })}
```